.travis.yml

```diff
@@ -17,22 +30,25 @@ sudo: required
 
 # We need PostgreSQL 10 installed
 addons:
-  postgresql: "10"
+  postgresql: 10
   apt:
     packages:
+      - jq
+      - curl
       - postgresql-10
       - postgresql-client-10
 
 # Environment Vars
 env:
   global:
+  - PR_AUTOBUILD: "true" # (example, enables automatic PR build when present)
   - PATH=$HOME/.local/bin:$PATH
-  - AWS_DEFAULT_REGION: us-east-1
-  - AWS_BUCKET_BACKUPS: joplin-austin-gov-archive
-  - JOPLIN_TARGET: joplin-personal
   - JOPLIN_IMAGE_NAME: joplin-base
+  # HEROKU_API_KEY="..."
```

Comment:
Unfortunate thing here, if we add another secure key, then these comments will immediately disappear. Not a real problem, just a good thing to note.

Reply:
Maybe these should be documented in the Readme file